### ISSUE TITLE
prefs are not applied? [solved: whole sections like 4600 are wrapped in comment syntax]

### ISSUE THREAD

**Issue by jjustauser** [ghacksuserjs/ghacks-user.js#584]
I don't know how relavant that is.
When I checked some values of the userjs with the actual values in about:config, (FF63)
I noticed that the value in user_pref("dom.enable_resource_timing", false);
but in about:config the value is dom.enable_resource_timing", true

the same with "media.video_stats.enabled"

**Comment by Thorin-Oakenpants** [comment]
the user.js master template turns on a pref called privacy.resistFingerprinting. It's pref number 4501. We call it RFP for short/

```js
user_pref("privacy.resistFingerprinting", true);
```

If you read the description for section 4500 (just the top part), you will see this
```
/*** [SECTION 4500]: RFP (RESIST FINGERPRINTING)
   This master switch will be used for a wide range of items, many of which will
   **override** existing prefs from FF55+, often providing a **better** solution

   IMPORTANT: As existing prefs become redundant, and some of them WILL interfere
   with how RFP works, they will be moved to section 4600 and made inactive
```

The two prefs you are talking about are in that **4600** section and have been made inactive (i.e commented out).

1. This is because the privacy or fingerprinting or tracking concerns those prefs in 4600 section are **now covered by RFP and unnecessary**
2. Additionally, RFP likes to mitigate the damage where possible, but let the feature still work - and that's **better compatibility** than just disabling it.
3. And, those prefs in section 4600, actually **still** affect your fingerprint if they are applied. So if you didn't have them inactive, then you would mess up your fingerprint.

The actual 4600 section is formatted slightly different to the other sections. The whole section is wrapped in a comment bracket - so it LOOKS like it's not commented out (in Notepad, but not in an cool IDE like Notepad++)

here is an example

this is the pref (2 lines from the user.js)
```js
// 4602: [2411] disable resource/navigation timing
user_pref("dom.enable_resource_timing", false);
```

here it is when I expand it to show you the top and bottom parts that wrap it
```js
/* [SETUP-non-RFP] Non-RFP users replace the * with a slash on this line to enable these

// 4602: [2411] disable resource/navigation timing
user_pref("dom.enable_resource_timing", false);

// ***/
```

and if I do as that line says (i.e if I do not use RFP but I still want protection, and I change that one character, this is what I get
```js
// [SETUP-non-RFP] Non-RFP users replace the * with a slash on this line to enable these

// 4602: [2411] disable resource/navigation timing
user_pref("dom.enable_resource_timing", false);

// ***/

```

---

So long story short. We use RFP by default, so the items in 4600 are commented out **as a group**, but if users want to not use RFP, they can change one character and enable section 4600 to act as if it wasn't commented out - does that all make sense to you now?
